mvp_example/src/main/java/com/distillery/android/blueprints/mvp/adapter/ToDoListAdapter.kt

```diff
@@ -0,0 +1,57 @@
+package com.distillery.android.blueprints.mvp.adapter
+
+import android.graphics.Paint.STRIKE_THRU_TEXT_FLAG
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.distillery.android.ui.databinding.ItemTodoBinding
+import com.distillery.android.domain.models.ToDoModel
+
+class ToDoListAdapter(
+    private val onDeleteClickListener: (toDoModel: ToDoModel) -> Unit,
+    private val onCompleteClickListener: (toDoModel: ToDoModel) -> Boolean
+) :
+        ListAdapter<ToDoModel, ToDoListAdapter.ToDoViewHolder>(ToDoListDiffCallback()) {
```

Comment:
Formatting is broken here.
Also, minor: `ToDoViewHolder` can be imported to shorten the line:
```suggestion
) : ListAdapter<ToDoModel, ToDoViewHolder>(ToDoListDiffCallback()) {
```

Reply:
It cannot be imported to shorten the line because it's an inner class.

Reply:
@sasfmlzr there is no such restriction. Here is an example
```kotlin
import com.test.Test2.InnerClass

abstract class Test1<T>

class Test2 : Test1<InnerClass>() {
    class InnerClass
}
```